docs/triggers/event-triggers.md

```diff
@@ -0,0 +1,586 @@
+## Event Triggers
+
+An Event Trigger is a Trigger that activates when a specific event occurs within the Slack client. Event Triggers can activate on either workspace,
+or channel level events, and each specific event type has its own required object parameters that need to be filled. An event Trigger at the highest
+level includes the common Trigger parameters along with a required input parameter and an event parameter: 
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| inputs          | Yes            | What inputs (defined in the manifest) are passed to the Trigger      |
```

Comment:
`inputs` is part of the common structure and is required if the Workflow requires inputs
```suggestion
```